app/adapters/request-config.js

```diff
@@ -0,0 +1,33 @@
+import AdapterError from '@ember-data/adapter/error';
+import V3Adapter from 'travis/adapters/v3';
+
+function parseJSON(string) {
+  try {
+    return JSON.parse(string);
+  } catch (e) {
+    return {};
+  }
+}
```

Comment:
that one doesn't exist on this branch, yet. i'll have to rebase to master.

Reply:
May be not rebase but just merge? Rebase here will likely be a nightmare 😅

Reply:
just rebased. was pretty straight forward :)

done: https://github.com/travis-ci/travis-web/pull/2399/commits/3aeacafa9d09f100c8f9e64a2e28e703328e739e

Reply:
Nice! Thank you 👍 Although, in your current use of the helper method it'll still break, because you didn't provide the default value to it: https://github.com/travis-ci/travis-web/commit/3aeacafa9d09f100c8f9e64a2e28e703328e739e#diff-4bf303175e8c929ff1750f67d6e9f125R13

It should be like this:
```js
let error = parseWithDefault(body.error_message, {}).error || {};
```
Or, with a little bit of javascript-fu 😄:
```js
let { error = {} } = parseWithDefault(body.error_message, {});
```
whatever you like more ;)